pkgs/development/python-modules/xstatic-font-awesome/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "XStatic-Font-Awesome";
```

```suggestion
  pname = "xstatic-font-awesome";
```
attr and pname should match most of the time.